clang/lib/Format/Format.cpp

```diff
@@ -586,7 +586,8 @@ template <>
 struct ScalarEnumerationTraits<FormatStyle::SeparateDefinitionStyle> {
   static void enumeration(IO &IO, FormatStyle::SeparateDefinitionStyle &Value) {
     IO.enumCase(Value, "Leave", FormatStyle::SDS_Leave);
-    IO.enumCase(Value, "Always", FormatStyle::SDS_Always);
+    IO.enumCase(Value, "One", FormatStyle::SDS_One);
+    IO.enumCase(Value, "Two", FormatStyle::SDS_Two);
     IO.enumCase(Value, "Never", FormatStyle::SDS_Never);
   }
```

Comment:
Does it need to be a spearate enum value or can it be handled as `SDS_One`?

Reply:
It will be `SDS_One`, it's just for the `.clang-format` file out there, which contain "Always". We provide compatibility for the config files, the users of libformat are not so lucky.
You'll find plenty of examples in other enums.